project/AppDependencies.scala

```diff
@@ -8,19 +8,19 @@ private object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "http-caching-client" % "9.5.0-play-28",
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.24.0",
+    "uk.gov.hmrc" %% "http-caching-client" % "9.6.0-play-28",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "6.3.0",
     "uk.gov.hmrc" %% "play-language" % "5.1.0-play-28",
-    "uk.gov.hmrc" %% "tax-year" % "1.3.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.13.0-play-28"
+    "uk.gov.hmrc" %% "tax-year" % "3.0.0",
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.22.0-play-28"
   )
 
   def testCommon(): Seq[ModuleID] = {
     val scope: String = "test,it"
     Seq(
       "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0"             % scope,
       "org.jsoup"              % "jsoup"               % "1.14.3"            % scope,
-      "uk.gov.hmrc" %% "bootstrap-test-play-28" % "5.24.0"  % scope,
+      "uk.gov.hmrc" %% "bootstrap-test-play-28" % "6.3.0"  % scope,
       "com.typesafe.play"      %% "play-test"          % PlayVersion.current % scope,
       "org.scalacheck"         %% "scalacheck"         % "1.15.4"            % scope,
```

Comment:
scalacheck can go to 1.16.0 and jackson-module-scala to 2.13.3.
scalaVersion in build.sbt can go to 2.12.15